pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -70,6 +72,7 @@
 import org.apache.pulsar.client.util.MessageIdUtils;
 import org.apache.pulsar.common.naming.TopicName;
 import org.apache.pulsar.common.util.FutureUtil;
+import org.bouncycastle.util.encoders.Hex;
```

Comment:
use org.apache.commons.codec.binary.Hex instead?

Reply:
Hi @lhotari,
 Do we need to encode/decode headers with Hex?
 Or String to byte conversion will suffice here?
 Please confirm so that i will add the header encoding accordingly.
 As of now, i just removed hex encoding and decoding and used bytes to produce and consume